packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -728,14 +728,16 @@ function updateOffscreenComponent(
       // Push the lanes that were skipped when we bailed out.
       const subtreeRenderLanes =
         prevState !== null ? prevState.baseLanes : renderLanes;
+      let prevCachePool = null;
       if (enableCache && current !== null) {
         // If the render that spawned this one accessed the cache pool, resume
         // using the same cache. Unless the parent changed, since that means
         // there was a refresh.
-        const prevCachePool = prevState !== null ? prevState.cachePool : null;
-        pushTransition(workInProgress, prevCachePool);
+        prevCachePool = prevState !== null ? prevState.cachePool : null;
       }
 
+      pushTransition(workInProgress, prevCachePool, null);
```

Comment:
Nit: Add a comment here to remind us to consider how transitions should work for Offscreen. Like maybe at some point we'll need a way to attribute pre-rendering work to the transition that spawned it.

> TODO: Consider if and how Offscreen pre-rendering should be attributed to the transition that spawned it